__tests__/utils/string.test.ts

```diff
@@ -0,0 +1,10 @@
+describe('utils/string', () => {
```

Comment:
please add the another case like:
- https://example.com
- http://example.com
- http://127.0.0.1:3002
- https://example.com/
- https://www.example.com

Reply:
fix on last commit